src/main/java/laura/Parser.java

```diff
@@ -0,0 +1,112 @@
+package laura;
+
+import laura.exception.LauraException;
+import laura.task.DeadlineTask;
+import laura.task.EventTask;
+import laura.task.ToDoTask;
+
+import java.util.Scanner;
+
+/**
+ * Deals with command handling for input
+ */
+public class Parser {
+    /** Scanner object to scan for input */
+    private final Scanner scanner;
+    /** Whether the user has given the command to exit */
+    private boolean exit;
```

Comment:
Maybe you can phrase this as a boolean instead?
```suggestion
    private boolean shouldExit;
```